doc/source/notebooks/simple_regression.py

```diff
@@ -38,12 +38,13 @@ def getSamples( m ):
     samples = m.sample(100, epsilon = 0.1)
     return samples
 
-def runExperiments(plotting=True,outputGraphs=False):
+def runExperiments(sampling=True,outputGraphs=False):
```

Comment:
Not sure how this crept in actually, I think this is the result of a merge rather than a conscious decision on my part?